pkgs/development/python-modules/numba/default.nix

```diff
@@ -12,29 +12,39 @@
 }:
 
 buildPythonPackage rec {
-  version = "0.53.0";
+  version = "0.54.1";
   pname = "numba";
-  # uses f-strings, python 3.9 is not yet supported
-  disabled = pythonOlder "3.6" || pythonAtLeast "3.9";
+  disabled = pythonOlder "3.6" || pythonAtLeast "3.10";
```

Comment:
AVA is built on python 3.9. I have not tested3.10. f-strings came way before 3.9 iirc around 3.5/3.6. Do we need py at least at ver 3.10?

Reply:
Yeah, this is also cherry-picked from upstream. The reason we do this is so we can drop the patch when we upgrade to the release that has the version we want.